### PR TITLE
Added TLS connection support for openldap collector

### PR DESCRIPTION
**Pull request by ekartsonakis**
##### Summary
Added support to make secure LDAP TLS connections. User can also choose to skip certificate check in case self signed ones are used.

Fixes: #5783

### CODE REVIEW

**Comment by ilyam8** [comment]
Hey @tctovsli could you please test the PR?

**Comment by ilyam8** [comment]
@ekartsonakis gj 👍 have you tested it?

And please sigh the CLA

**Comment by ekartsonakis** [comment]
@ilyam8 yes I tested with all possible options. 
OK I just signed.

**Comment by ilyam8** [comment]
@ekartsonakis i see CLA is not signed. The commit is from `Emmanouil Kartsonakis', this is the problem.

File: collectors/python.d.plugin/openldap/openldap.conf [hunk 1]
```diff
@@ -67,6 +67,8 @@ update_every: 10
 
 #username : "cn=admin,dc=example,dc=com"   # The bind user with right to access monitor statistics
 #password : "yourpass"                     # The password for the binded user
-#server   : 'localhost'                    # The listening address of the LDAP server
-#port     : 389                            # The listening port of the LDAP server
-#timeout  : 1                              # Seconds to timeout if no connection exists
\ No newline at end of file
+#server   : 'localhost'                    # The listening address of the LDAP server. In case of TLS, use the hostname which the certificate is published for.
+#port     : 389                            # The listening port of the LDAP server. Change to 636 port in case of TLS connection
+#tls      : False                          # Make True if a TLS connection is used
```
[the remark this answers is not in the document]
Comment (ekartsonakis) [reply collectors/python.d.plugin/openldap/openldap.conf]: yes, why not. The more clear the better! :)

**Comment by ilyam8** [comment]
@ekartsonakis i am not able to merge w/o the CLA being signed

**Comment by ekartsonakis** [comment]
ok signed again.

**Comment by ilyam8** [comment]
@ekartsonakis 

check this

your prev PR commits
https://github.com/netdata/netdata/pull/4513/commits

this one
https://github.com/netdata/netdata/pull/5827/commits

different accounts

**Comment by ilyam8** [comment]
> Emmanouil Kartsonakis seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account.

**Comment by ilyam8** [comment]
closing in favor of #5859

**Comment by ekartsonakis** [comment]
oohhhh...yes sorry, I used a wrong git config :( sorry. I rebased and created [another pull request](https://github.com/netdata/netdata/pull/5859). So I am closing this one.

**Pull request closed without merging**